### PR TITLE
dbms.killQueries now shows if it failed to kill a query

### PR DESCRIPTION
**Pull request by eebus**

### CODE REVIEW

File: community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java [hunk 1]
```diff
@@ -567,6 +567,10 @@
     public static final Setting<String> procedure_unrestricted =
             setting( "dbms.security.procedures.unrestricted", Settings.STRING, "" );
 
+    @Description("If dbms.killQueries give a verbose output, with information about whitch querys where not found." )
```
Comment (pontusmelke) [inline community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java]: Do we really need a setting for this, couldn't it just always be verbose?

Comment (pontusmelke) [reply community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java]: `whitch` -> `which`

File: enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java [hunk 2]
```diff
@@ -312,17 +314,21 @@ private boolean isAdminProcedure( String procedureName )
...
         securityContext.assertCredentialsNotExpired();
         try
         {
             long queryId = fromExternalString( idText ).kernelQueryId();
 
-            return getActiveTransactions( tx -> executingQueriesWithId( queryId, tx ) )
-                .map( catchThrown( InvalidArgumentsException.class, this::killQueryTransaction ) );
-         }
+            Set<Pair<KernelTransactionHandle,ExecutingQuery>> querys = getActiveTransactions( tx -> executingQueriesWithId( queryId, tx ) ).collect( toSet() );
+            Config configs = resolver.resolveDependency( Config.class );
```
Comment (pontusmelke) [inline enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java]: We only need to find the setting once, right? Do not need to do it for each call to `killQuery`?

File: enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java [hunk 3]
```diff
@@ -332,20 +338,30 @@ private boolean isAdminProcedure( String procedureName )
...
-                .collect( toSet() );
 
-            return getActiveTransactions( tx -> executingQueriesWithIds( queryIds, tx ) )
-                .map( catchThrown( InvalidArgumentsException.class, this::killQueryTransaction ) );
+            Set<Long> queryIds = idTexts.stream().map( catchThrown( InvalidArgumentsException.class, QueryId::fromExternalString ) ).map(
+                    catchThrown( InvalidArgumentsException.class, QueryId::kernelQueryId ) ).collect( toSet() );
+
+            Set<QueryTerminationResult> terminatedQuerys = getActiveTransactions( tx -> executingQueriesWithIds( queryIds, tx ) ).map(
+                    catchThrown( InvalidArgumentsException.class, this::killQueryTransaction ) ).collect( toSet() );
+            Config configs = resolver.resolveDependency( Config.class );
```
Comment (pontusmelke) [inline enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java]: Same here, this needs could be done once

File: community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java (hunk 1 above)
Comment (eebus) [reply community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java]: The intent is to not change old behavior in a patch release for 3.2. In the forward merge, this config should be flipped to support the verbose as the default.

File: community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java [hunk 4]
```diff
@@ -567,6 +567,10 @@
     public static final Setting<String> procedure_unrestricted =
             setting( "dbms.security.procedures.unrestricted", Settings.STRING, "" );
 
+    @Description("If dbms.killQueries give a verbose output, with information about which querys where not found." )
```
Comment (OliviaYtterbrink) [inline community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java]: This description could be better, maybe `Specifies whether or not dbms.killQueries produces a verbose output, with information about which queries were not found`?

**Review by OliviaYtterbrink — APPROVED** [review]

**Pull request merged**